docs/workstation_release_management.rst

```diff
@@ -14,176 +11,99 @@ SecureDrop Workstation code spans across several repositories:
-6. Set ``PKG_DIR`` to point to the tarball you wish to package, and
-   ``PKG_VERSION`` to the version you wish to package, then run the
-   relevant makefile target in the
-   `securedrop-debian-packaging <https://github.com/freedomofpress/securedrop-debian-packaging>`__
-   repository. For example to build version 0.1.1 of the
-   ``securedrop-client``:
-
-``$ PKG_VERSION=0.1.1 PKG_PATH=tarballs/securedrop-client-0.1.1.tar.gz make securedrop-client``
-
-6.  Upload build logs in the
-    `build-logs <https://github.com/freedomofpress/build-logs>`__
-    repository in the workstation directory. Ensure that the sha256sum
-    of the built package is included in the build log.
-7.  Next, add the package via PR to the private
-    `securedrop-debian-packages-lfs <https://github.com/freedomofpress/securedrop-debian-packages-lfs>`__
-    repository.
-8.  Regenerate reprepro repository metadata using the script in that
-    repository: ``./tools/publish``. When you inspect the diff, you’ll
-    notice that the previous version of the subproject will no longer be
-    served. This is expected.
-9.  Copy the ``Release`` file to signing environment.
-10. Verify integrity of ``Release`` file.
-11. Sign the Release file
-    ``gpg --armor --detach-sign --output Release.gpg Release``
-12. Copy the detached signature into your working directory and commit
-    along with the new package(s), and the modified repository metadata.
-13. Open a PR for review.
-14. Upon merge to master, ensure that changes deploy to
-    ``apt.freedom.press`` without issue.
-
-RPM package
-~~~~~~~~~~~
+Releasing a Debian package can take some time because it entails at least two package builds, multiple deployments to our different apt repositories, quality assurance testing, two signing ceremonies, and stakeholder communications.
+
+Releasing an rc package is the first step before you begin QA or any signing ceremonies. Even when you are releasing a hotfix, it is recommended to start with an rc package to defer needing the Securedrop release key until the day of release when you are ready to deploy the package to production.
+
+On release day (usually at least a couple weeks after releasing your first rc package), you (or another maintainer) will need to have access to the SecureDrop release key to sign both a new release tag (in order to build a production version of the package, which you will verify has a hash that matches the rc package hash that was approved during QA) and updated Debian Release file for the production apt repository. You will also need at least one other maintainer to review your PRs and perform the final QA checks once the package lands on https://apt-qa.freedom.press and later on https://apt.freedom.press.
+
+## Step 1: Create an rc tag.
+
+1. Create a release branch in the repo of the component you want to release.
+2. Push a changelog commit.
+3. Push an rc tag on your new commit. We will be building from this tag in the next step.
```

Comment:
If there are instructions on what the rc tags should be created (naming conventions...), I think this would be a good place to link to them.